src/admin.py

```diff
@@ -339,20 +340,111 @@ def list_workflow_templates():
     print(table)
 
 
+# @app.command()
+# def purge_deleted_filesOFF(
```

Comment:
Is this supposed to be all commented out?

Reply:
ooops, my bad. Good catch :) Removed now.